corehq/pillows/xform.py

```diff
@@ -143,9 +143,22 @@ def get_xform_to_elasticsearch_pillow(pillow_id='XFormToElasticsearchPillow', nu
     Processors:
       - :py:class:`pillowtop.processors.elastic.ElasticProcessor`
     """
+    index_info = XFORM_INDEX_INFO
     # todo; To remove after full rollout of https://github.com/dimagi/commcare-hq/pull/21329/
-    assert pillow_id == 'XFormToElasticsearchPillow', 'Pillow ID is not allowed to change'
-    checkpoint = get_checkpoint_for_elasticsearch_pillow(pillow_id, XFORM_INDEX_INFO, FORM_TOPICS)
+    if 'index_name' in kwargs:
+        raw_info = XFORM_INDEX_INFO.to_json()
+        raw_info.pop("meta")
+        index_info = ElasticsearchIndexInfo.wrap(raw_info)
```

Comment:
Is the stickler here right that this needs to be imported?

Reply:
Apparently